quadlet/minio-prometheus/prometheus.container

```diff
@@ -8,7 +8,7 @@ Image=docker.io/prom/prometheus:latest
 Network=bridge
 Network=prometheus.network
 PublishPort=9090:9090
-Volume=./prometheus.yml:/etc/prometheus/prometheus.yml:ro
+Volume=./prometheus.yml:/etc/prometheus/prometheus.yml:Z
```

Comment:
`:ro` is required when running in `/etc/containers/systemd/users` but `:Z` must be used in the home directory.  It's a bit strange to me, but I'm sure `:ro` doesn't work in $HOME for a good reason.

Reply:
It's SELinux that's blocking the access and for which you need to set `:Z`

Reply:
You can set both :ro,Z

Reply:
@p5 Please see @rhatdan's comment. What would you like to do here?

Reply:
Apologies, I was getting myself in a muddle since I store quadlets in /etc/containers, where `:z` doesn't work with a user service.  Since this example is written assuming users will store the file in their home directory, I have set the config file as readonly and confirmed it works.